megamek/src/megamek/common/verifier/TestTank.java

```diff
@@ -536,6 +536,10 @@ public static boolean legalForMotiveType(EquipmentType eq, EntityMovementMode mo
             if (eq.hasFlag(MiscType.F_EXTERNAL_STORES_HARDPOINT)) {
                 return mode.equals(EntityMovementMode.AERODYNE);
             }
+            if (eq.hasFlag(MiscType.F_MAST_MOUNT)
+                || (eq.hasFlag(MiscType.F_MASC) && eq.hasFlag(MiscType.F_VTOL_EQUIPMENT))) {
```

Comment:
VTOLs can mount MASC?

Reply:
In MiscType the VTOL Jet Booster has the MASC flag for some reason. That's the only reason I can think of that it was included. As I said in my bug post, or perhaps to Hammer in discord, there are a lot of odd type flags in some of the equipment.

Reply:
The F_MASC flag marks equipment that increases speed. MASC, superchargers, VTOL jet boosters, and battlearmor myomer boosters all have it. The only equipment that has both the F_MASC and F_VTOL_EQUIPMENT flags is the VTOL jet booster.